config/bref.php

```diff
@@ -30,4 +30,16 @@
 
     'request_context' => false,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Jobs Logging
+    |--------------------------------------------------------------------------
+    |
+    | Here you can enable detailed logging of every job execution for debugging
+    | purposes. Note, that enabling this, will result in a high volume of log
+    | entries.
+    |
+    */
+
+    'jobs_logs' => false,
```

Comment:
```suggestion
    'log_jobs' => false,
```

as per https://github.com/brefphp/laravel-bridge/issues/146#issuecomment-2370584166

Reply:
sorry for the typo - fixed it